tests/util/common.ts

```diff
@@ -1,4 +1,10 @@
 import { BN } from "@polkadot/util";
+import Bottleneck from "bottleneck";
+
+export function rateLimiter() {
+  const settings = process.env.CI === "true" ? {} : { maxConcurrent: 10, minTime: 150 };
```

Comment:
```suggestion
  const settings = process.env.SKIP_RATE_LIMITER === "true" ? {} : { maxConcurrent: 10, minTime: 150 };
```
Please find a more explicit anme for this env var, `CI` is tto short and also wrong because we not setting this env var on our github workflows